control-plane-operator/controllers/hostedcontrolplane/kas/service.go

```diff
@@ -96,5 +96,6 @@ func ReconcilePrivateService(svc *corev1.Service, owner *metav1.OwnerReference)
 }
 
 func ReconcilePrivateServiceStatus(svc *corev1.Service, mgmtBaseDomain string) (host string, port int32, err error) {
-	return fmt.Sprintf("api-%s.%s", svc.GetNamespace(), mgmtBaseDomain), 6443, nil
+	//return fmt.Sprintf("api-%s.%s", svc.GetNamespace(), mgmtBaseDomain), 6443, nil
+	return fmt.Sprintf("kube-apiserver.%s.svc", svc.Namespace), 6443, nil
```

Comment:
nit: fmt.Sprintf("%s.%s.svc", svc.Name, svc.Namespace) 
and remove the commented out line above